config.toml

```diff
@@ -48,4 +49,16 @@ publish_hold = [
     "/topics/internals/",
     "/topics/protocol/",
     "/topics/rdd/"
+]
+
+recommended_clients_paths = [
+    "/python/github.com/AWS/valkey-GLIDE.json",
```

Comment:
this probably is more relevant for the docs repo but GLIDE is a valkey project, not an AWS project, so the path should be different.

Reply:
Fixed in the docs repo.